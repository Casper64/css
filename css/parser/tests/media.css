@media only, ,print, {}

@media (max-width: 600px) {
    .test {
        color: green;
    }

.other {
    color: red;
}

