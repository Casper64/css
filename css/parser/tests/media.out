css/parser/tests/media.css:1:14: warning: expecting a media query or "{"
    1 | @media only, ,print, {}
      |              ^
    2 | 
    3 | @media (max-width: 600px) {
css/parser/tests/media.css:1:20: warning: expecting a media query or "{"
    1 | @media only, ,print, {}
      |                    ^
    2 | 
    3 | @media (max-width: 600px) {
css/parser/tests/media.css:12:1: error: expecting "}". Did you forget to close the previous block?
   10 | }
   11 |